packages/types/src/index.ts

```diff
@@ -275,7 +275,7 @@ export interface SectionBlock extends Block {
   type: 'section';
   text?: PlainTextElement | MrkdwnElement; // either this or fields must be defined
   fields?: (PlainTextElement | MrkdwnElement)[]; // either this or text must be defined
-  accessory?: Button | Overflow | Datepicker | Select | MultiSelect | Action | ImageElement | RadioButtons;
+  accessory?: Button | Overflow | Datepicker | Select | MultiSelect | Action | ImageElement | RadioButtons | Checkboxes;
```

Comment:
Just for future reference: A `checkboxes` element in a section block is not allowed in messages. That said, considering the lack of ways to distinguish if the container of blocks is either a message or a view, I think there is no better way than just adding `Checkboxes` type for modals and Home tabs.

Reply:
> A checkboxes element in a section block is not allowed in messages.

@seratch Ah, right. But, yeah, I also think that it would be better to choose a benefit to add `Checkboxes` type for both modal and home tab at this time.